addons/mail/static/src/components/messaging_menu/messaging_menu.js

```diff
@@ -164,6 +166,8 @@ class MessagingMenu extends Component {
             return;
         }
         this.messagingMenu.toggleOpen();
+        const preventScroll = this.messagingMenu.isOpen && this.env.messaging.device.isMobile;
+        document.querySelector('body').style.position =  preventScroll ? 'fixed' : this._bodyPosition;
```

Comment:
I think `fixed` makes it scroll to the top automatically which might not be desirable. I think the better way for this would be `overflow: hidden` to just remove the scrollbar.

But as you imagine, it does not look like a good idea to have an element control `body` directly like that.

By the way there are probably existing ways to do such things to body, maybe in dialog or blockui?